static_src/components/service_instance_list_panel.jsx

```diff
@@ -0,0 +1,54 @@
+
+import style from 'cloudgov-style';
+import React from 'react';
+
+import PanelRow from './panel_row.jsx';
+import ServiceInstance from './service_instance.jsx';
+
+import createStyler from '../util/create_styler';
+
+const propTypes = {
+  serviceInstances: React.PropTypes.array,
```

Comment:
Should this sort of information come straight from the stores? Right now [`<RouteList />`](https://github.com/18F/cg-dashboard/blob/master/static_src/components/route_list.jsx#L5-L15) uses a bunch of stores so so that the component is isolated and doesn't need many props.

I think this is nice because that keeps the app container component a bit cleaner and doesn't have a bunch of logic about data filtering/processing to hand off to child components.

Of course, the benefit of having a prop is that the API is super clear and its obvious how to use the the component or have it display different data.

Maybe we can have these two methods for now and see how most of the other panel components shake out? I'd think that we'd want them all to function in pretty similar manners for ease of comprehension.